src/libs/antares/study/include/antares/study/parts/hydro/container.h

```diff
@@ -28,6 +28,63 @@
 
 namespace Antares::Data
 {
+
+enum dayYearCount
+{
+    //! The maximum number of days in a year
+    dayYearCount = 366
+};
+
+struct HydroDaily
+{
+    //! Net demand, for each day of the year, for each area
+    std::array<double, dayYearCount> DLN;
+    //! Daily local effective load
+    std::array<double, dayYearCount> DLE;
+};
+
+struct HydroMonthly
+{
+    //! Monthly local effective demand
+    std::array<double, 12> MLE;
+    //! Monthly optimal generation
+    std::array<double, 12> MOG;
+    //! Monthly optimal level
+    std::array<double, 12> MOL;
+    //! Monthly target generations
+    std::array<double, 12> MTG;
+};
+
+// TODO Rename
+struct HydroSpecific
+{
+    HydroDaily daily;
```

Comment:
You may want to use `double`s in `HydroDaily`, and an `std::array<HydroDaily, 12>` here, so you have 1 array instead of multiple. The same goes for monthly data.